whitelabelco/sofo-foods
# Cp/feat/requested updates

### Description 💻

Resolves: #225 #227 #228 

### Implementation 💡

- swap content on the About Us page: Mission should go in first content section and story in second
- update button text on Brands page: should be OUR PRODUCTS and PLACE AN ORDER
- add new product images to recently added products: Spices, Seafood, Pizza Toppings, Pasta, Oils, Desserts, Mushrooms Olives, Condiments


### How to test 🔍

- `npm i`
- `npm run dev`
- navigate to `http://localhost:3000/`
- click on About Us
- confirm content is in correct sections
- click on Brands
- confirm button text has been update
- click on Products
- confirm new images on product grid match the corresponding product
- click on the individual product pages and confirm hero image has been updated also

### Screenshots/Demos 📷

### Checklist ✅
